### PR TITLE
Launcher Script

### PR DESCRIPTION
**Pull request by kealdor**
Launches MTGA, then MTGA Tracker. if MTGA isn't running it closes Tracker

### CODE REVIEW

**Comment by kealdor** [comment]
thanks but i did see i had a error way in the bottom i didnt 100% notice earlier but i fixed it locally but im trying to get it to fix on here now as well

**Comment by kealdor** [comment]
fixed with online editor

File: scripts/mtga_and_tracker_launcher.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+__author__ = "Kealdor"
+__date__ = "$May 28, 2018 8:27:00 PM$"
+
+from sys import platform as _platform
+import subprocess
+
+#Set path to MTGA
+MTGA_Path = ''
```
Comment (shawkinsl) [inline scripts/mtga_and_tracker_launcher.py]: Minor: per pep8, variable names should be `all_lower_with_underscores`

File: scripts/mtga_and_tracker_launcher.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   MTGA_Tracker_Path = os.getenv("USERPROFILE") + "\\AppData\\Local\\mtgatracker\\MTGATracker.exe"
+   
+#Run The Exe's
+os.system(MTGA_Path)
+os.system(MTGA_Tracker_Path)
+
+#Close The Exe's when needed
+n=0# number of instances of the program running 
+
+prog = [line.split() for line in subprocess.check_output("tasklist").splitlines()]
```
Comment (shawkinsl) [inline scripts/mtga_and_tracker_launcher.py]: `tasklist` isn't going to be cross-platform, but we can address this later

File: scripts/mtga_and_tracker_launcher.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+prog.pop(e) for e in [0,1,2]]
+
```
Comment (shawkinsl) [inline scripts/mtga_and_tracker_launcher.py]: `[0,1,2]` should be `range(3)`

File: scripts/mtga_and_tracker_launcher.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+for task in prog:
+        if task[0]=="MTGA.exe":
```
Comment (shawkinsl) [inline scripts/mtga_and_tracker_launcher.py]: Minor: overindendation

File: scripts/mtga_and_tracker_launcher.py (hunk 2 above)
Comment (kealdor) [reply scripts/mtga_and_tracker_launcher.py]: true but for the time being tracker just runs on wndows? but unoffically MTGA runs on linux already

**Pull request closed without merging**